doc/guides/managing-a-release/README.md

```diff
@@ -0,0 +1,299 @@
+# Contents
+
+1. [Checklist](https://github.com/RIOT-OS/RIOT/wiki/Managing-a-Release#1-checklist)
+2. [Preparation](https://github.com/RIOT-OS/RIOT/wiki/Managing-a-Release#2-preparation)
+3. [Feature Freeze and Testing](https://github.com/RIOT-OS/RIOT/wiki/Managing-a-Release#3-feature-freeze-and-testing)
+4. [Drafting Release Notes](https://github.com/RIOT-OS/RIOT/wiki/Managing-a-Release#4-Drafting-Release-Notes)
+5. [Actual Release](https://github.com/RIOT-OS/RIOT/wiki/Managing-a-Release#5-actual-release)
+6. [Other Repositories](https://github.com/RIOT-OS/RIOT/wiki/Managing-a-Release#6-Other-repositories)
+7. [Important Fixes after Release](https://github.com/RIOT-OS/RIOT/wiki/Managing-a-Release#7-important-fixes-after-release)
+8. [Resources](https://github.com/RIOT-OS/RIOT/wiki/Managing-a-Release#8-resources)
+9. [Email templates](https://github.com/RIOT-OS/RIOT/wiki/Managing-a-Release#9-email-templates)
+
+# 1. Checklist
+
+**Preparation**
+- [ ] Read “Managing a release” wiki
+- [ ] Gather improvements and tips from previous release manager
+- [ ] Decide and announce feature freeze dates
+- [ ] Create milestone label :scream_cat:, and label PRs that have been already merged
+- [ ] Gather features to push from contributors, and apply milestone label
+- [ ] Go through list of deprecated features and make sure pending removals are applied
+- [ ] Familiarise with release specs and tests, incl open issues, deprecations, api changes, and PRs
+
+**Soft feature freeze**
+- [ ] Send soft feature freeze email
+- [ ] Check that there are no pending Rust changes (`find -name Cargo.toml -exec cargo update --manifest-path "{}" --package riot-wrappers --package riot-sys ";"` should be a no-op), otherwise poke Rust using maintainers
+- [ ] Familiarise with nearly-merged, high impact PRs, and contact the contributors to ask them to hold
+
+**Hard Feature Freeze**
+
+Do the below actions iteratively, generating new release candidates, until all release specs tests pass.
+- [ ] Generate branch and tag for release candidate
+- [ ] Add [branch protection rules](https://github.com/RIOT-OS/RIOT/settings/branches) for the release branch (if you don't have permissions ask an [admin](https://github.com/orgs/RIOT-OS/teams/admin) or [owner](https://github.com/orgs/RIOT-OS/teams/owners)). :warning: A new rule has to be added manually for each release, since the Github Merge Queue cannot be enabled for branches that are protected with a wildcard rule for some reason.
+- [ ] Open issue in release specs repo for release candidate
+- [ ] Send hard feature freeze email
+- [ ] Coordinate testing for release candidate
+- [ ] Coordinate bugfixing for release candidate on master
+- [ ] Backport bugfixes for release candidate to release branch
+
+**Releasing**
+- [ ] Create a GPG key and upload it to GitHub
+- [ ] Generate etherpad with list of new features, deprecations, api changes, and known issues, and share
```

Comment:
This is handled by the `riot-release-manager` script now:

`riot-release-manager set-milestones` adds all PRs merged since the last release to the current release milestone, then 

`riot-release-manager release-notes` generates the release notes from all PRs tagged with the release milestone